[2024-12-19T08:54:52.664+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T08:54:52.684+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.first_task scheduled__2022-12-27T00:00:00+00:00 [queued]>
[2024-12-19T08:54:52.704+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.first_task scheduled__2022-12-27T00:00:00+00:00 [queued]>
[2024-12-19T08:54:52.705+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-19T08:54:52.732+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): first_task> on 2022-12-27 00:00:00+00:00
[2024-12-19T08:54:52.750+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=6198) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T08:54:52.754+0000] {standard_task_runner.py:72} INFO - Started process 6209 to run task
[2024-12-19T08:54:52.751+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'first_task', 'scheduled__2022-12-27T00:00:00+00:00', '--job-id', '1383', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpx21ux4mq']
[2024-12-19T08:54:52.756+0000] {standard_task_runner.py:105} INFO - Job 1383: Subtask first_task
[2024-12-19T08:54:52.852+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v2.first_task scheduled__2022-12-27T00:00:00+00:00 [running]> on host 8d142e9c83a7
[2024-12-19T08:54:52.985+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mateo' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2022-12-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-12-27T00:00:00+00:00'
[2024-12-19T08:54:52.989+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T08:54:53.017+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-12-19T08:54:53.018+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task']
[2024-12-19T08:54:53.042+0000] {subprocess.py:99} INFO - Output:
[2024-12-19T08:54:53.046+0000] {subprocess.py:106} INFO - hello world, this is the first task
[2024-12-19T08:54:53.050+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-12-19T08:54:53.086+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T08:54:53.088+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=first_task, run_id=scheduled__2022-12-27T00:00:00+00:00, execution_date=20221227T000000, start_date=20241219T085452, end_date=20241219T085453
[2024-12-19T08:54:53.146+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-19T08:54:53.181+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-19T08:54:53.186+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
